### ISSUE TITLE
the alt modifier key is stuck after using alt+tab twice

### ISSUE THREAD

**Issue by soundmud** [pygame/pygame#436]
On Windows 10, if you use alt+tab twice, or if you start with a desktop shortcut with ctrl+alt+..., the modifier keys will stick, until you press and release them. Debian 9 doesn't have this problem.

This affects key.get_mods() and key.get_pressed(). Additionally, the next KEYDOWN event concerning a stuck modifier key will be ignored, probably because SDL assumes that the key is already pressed.

This bug seems related to this one:
https://discourse.libsdl.org/t/modifier-keys-pressed-during-initialization-stick/11736

To reproduce the bug, here is a script:
```
#! python2

# To test:
# - start this script, making sure the results are displayed in a console
# (on Windows, double-click on the script file)
# - press and release the left alt key
#   => the console should print the correct state
# - use alt+tab twice to switch to another window and back
#   => *** the left alt key is stuck on the pressed state
#      until the alt key is pressed and released again! ***
#
# Results:
# - on Windows 10 with pygame 1.9.1 and Python 2.7 => bug
# (note: no bug if you switch to another window by clicking on it)
# - tested on Debian 9 with pygame 1.9.1 and Python 2.7 => no bug

import pygame
import sys

print pygame.ver
print sys.version
pygame.init()
pygame.display.set_mode((200, 20))
while True:
    pygame.event.pump()
    if pygame.key.get_focused():
        print "left alt key:", pygame.key.get_mods() & pygame.KMOD_LALT
    else:
        print "unfocused (press alt+tab to switch back to the display window)"
    pygame.time.wait(100)
```

**Comment by MyreMylar** [comment]
This does not occur on windows 10 with Python 3.6 & Pygame 1.9.6 or with Python 3.8 and Pygame 2.0.0.dev7. Tested using original code ported to Python 3:

    #! python3
    
    # To test:
    # - start this script, making sure the results are displayed in a console
    # (on Windows, double-click on the script file)
    # - press and release the left alt key
    #   => the console should print the correct state
    # - use alt+tab twice to switch to another window and back
    #   => *** the left alt key is stuck on the pressed state
    #      until the alt key is pressed and released again! ***
    #
    # Results:
    # - on Windows 10 with pygame 1.9.1 and Python 2.7 => bug
    # (note: no bug if you switch to another window by clicking on it)
    # - tested on Debian 9 with pygame 1.9.1 and Python 2.7 => no bug
    
    import pygame
    import sys
    
    print(pygame.ver)
    print(sys.version)
    pygame.init()
    pygame.display.set_mode((200, 20))
    while True:
        pygame.event.pump()
        if pygame.key.get_focused():
            print("left alt key:", pygame.key.get_mods() & pygame.KMOD_LALT)
        else:
            print("unfocused (press alt+tab to switch back to the display window)")
        pygame.time.wait(100)

Since this no longer occurs and the original report was for Python 2, a now deceased programming language. I believe we should declare this a dead issue.

There is no good reason I can see to use Python 2 on Windows anymore. 

Recommend close.

**Comment by soundmud** [comment]
Tested on Windows 10 with pygame 1.9.6 and Python 2.7.9 => no bug.
Tested on Windows 10 with pygame 1.9.6 and Python 3.8.2 => no bug.
Tested on Windows 10 with pygame 1.9.1 and Python 2.7.9 => bug.
Tested on Windows 10 with pygame 1.9.2b1 and Python 2.7.9 => no bug.
So it seems that Windows 10 didn't change and that the bug was fixed by pygame 1.9.2.
That's great news. Thanks for the reply!
